tests/test_core.py

```diff
@@ -322,6 +322,10 @@ def test_sintvars(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_sintvars',
                                  force_c=True)
 
+  def test_int53(self):
+    self.emcc_args += ['-s', 'DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=[$convertI32PairToI53,$convertU32PairToI53,$readI53FromU64,$readI53FromI64,$writeI53ToI64,$writeI53ToI64Clamped,$writeI53ToU64Clamped,$writeI53ToI64Signaling,$writeI53ToU64Signaling]']
```

Comment:
Is the JS compiler not able to see references to these functions in the EM_ASM code?  I guess not.

Reply:
Yeah, EM_ASM and EM_JS do not have a __deps machinery to them.